.github/CONTRIBUTING.md

```diff
@@ -12,20 +12,20 @@ If you are submitting a bug report, please include the following:
 
 * When submitting *bug reports*, *paste your schema example* whenever possible. This will save us from having to ask you for it later.
 * When submitting *Pull Requests (PRs)*, you should request to merge your changes into our `develop` branch.
-* Our approach to testing is described in [spec/README.md](/spec/README.md).
+* Our approach to testing is described in [docs/TESTING.md](/docs/TESTING.md).
 * PRs without specs will not be merged anytime soon!
 
 ## Releasing new versions
 
 * Create a new entry in the [CHANGELOG](/CHANGELOG) file,
-* Re-build and re-release the online demo, see [`website-jsf` gh-pages branch](https://github.com/json-schema-faker/website-jsf/tree/gh-pages).
+* Re-build and re-release the online demo, see [`json-schema-faker` gh-pages branch](https://github.com/json-schema-faker/json-schema-faker/tree/gh-pages).
```

Comment:
It looks like the website was merged into this repository so I believe this change is correct.

Reply:
That's true, rebuilding the demo is not useful until you want to add/fix stuff there (and only if it's related/needed).

We should make such step optional, back to that time we probably wanted to include a built version on the website (not anymore thanks to unpkg).